examples/inverse/plot_gamma_map_inverse.py

```diff
@@ -57,10 +57,10 @@
 
 # Show the evoked response and the residual for gradiometers
 ylim = dict(grad=[-120, 120])
-evoked = mne.pick_types_evoked(evoked, meg='grad', exclude='bads')
+evoked.pick_types(meg='grad', exclude='bads')
 evoked.plot(titles=dict(grad='Evoked Response Gradiometers'), ylim=ylim,
             proj=True)
 
-residual = mne.pick_types_evoked(residual, meg='grad', exclude='bads')
+residual.pick_types(meg='grad', exclude='bads')
```

Comment:
@Eric89GXL why don't we add this everywhere. Would be lovely as an addition to pick/drop_channels. I would suggest to add this to the related mixin.